### ISSUE TITLE
Accessibility: more unlabeled buttons

### ISSUE THREAD

**Issue by shortwavesurfer2009** [hjiangsu/thunder#51]
**Description**  
The buttons in the screenshots below either are not read at all or read as "unlabeled"

**How to Reproduce**  
1. Go to System settings > Accessibility > talkback and turn it on
2. Open thunder
3. Tap these buttons with 1 finger

**Expected Behavior**  
These buttons should be labeled for use with screen readers

**Screenshots**  
![Screenshot_20230623-103122_thunder](https://github.com/hjiangsu/thunder/assets/116814522/9c4cdb40-8c95-4eaf-b928-0d484f2fb7ad)
![Screenshot_20230623-103136_thunder](https://github.com/hjiangsu/thunder/assets/116814522/046213ff-f7f8-4398-9dcb-9c5cc2c476b6)
![Screenshot_20230623-103143_thunder](https://github.com/hjiangsu/thunder/assets/116814522/7d101bcb-f2a1-475c-add0-c28fe114ccf5)
![Screenshot_20230623-103146_thunder](https://github.com/hjiangsu/thunder/assets/116814522/b6105c76-4e44-4443-835a-f1eaddc0d29d)
![Screenshot_20230623-103149_thunder](https://github.com/hjiangsu/thunder/assets/116814522/eb84032d-40ed-45e6-96e7-c9bd0d3afe93)
![Screenshot_20230623-103151_thunder](https://github.com/hjiangsu/thunder/assets/116814522/c412cfaa-99de-4083-81f4-136d649af1fe)
![Screenshot_20230623-103154_thunder](https://github.com/hjiangsu/thunder/assets/116814522/b4414ce1-463d-4be9-abe6-f131c15e5285)
![Screenshot_20230623-103204_thunder](https://github.com/hjiangsu/thunder/assets/116814522/0c50b525-9b93-4d6d-b2fa-e878e2c3d37f)
![Screenshot_20230623-103207_thunder](https://github.com/hjiangsu/thunder/assets/116814522/3158276a-94f7-49e7-8d85-0c8610e2f366)
![Screenshot_20230623-103220_thunder](https://github.com/hjiangsu/thunder/assets/116814522/7c21bae4-9a1a-4fa5-baea-917fa43675e2)
![Screenshot_20230623-103830_thunder](https://github.com/hjiangsu/thunder/assets/116814522/c7b2ec7b-c213-4497-b18e-a86b43aedb71)

**Device & App Version:**  
 - Device: [e.g. iPhone6] Motorola One 5G Ace
 - OS: [e.g. iOS8.1] Android 11
 - Version [e.g. 22] 0.2.1+5

**Comment by hjiangsu** [comment]
This should be done as well, and will come in the next release!

**Comment by shortwavesurfer2009** [comment]
Just tested with the prerelease found here https://github.com/hjiangsu/thunder/releases/tag/v0.2.1-prerelease%2B5 and this issue is fixed. Closing as complete.
